### PR TITLE
Ubuntu-specific installation instructions

### PR DESCRIPTION
**Pull request by sieira**
Adding an ubuntu-specific installation manual as suggested in [this jira thread](https://ci.open-paas.org/jira/browse/OR-1471?filter=-2)

Some of the steps contained in this manual may eventually be merged with the [general installation instructions](https://github.com/linagora/openpaas-esn/blob/master/README.md) in the README.md file at the root of openpaas-esn

### CODE REVIEW

**Comment by kewisch** [comment]
Hi Luis, lots of folks are on vacation right now, please be patient :-) I'd like someone from the team that actually uses Ubuntu to look over the instructions before we merge this.

File: doc/install-ubuntu.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+2. clone the repository
+
+        git clone https://ci.open-paas.org/stash/scm/or/rse.git
+
+3. Install ElasticSearch 1.3.5
+ Download and install ElasticSearch 1.3.5 [deb package](https://www.elastic.co/downloads/past-releases/elasticsearch-1-3-5) from elastic website
+
+        wget https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.3.5.deb
+        dpkg -i elasticsearch-1.3.5.deb
```
Comment (Agrumas) [inline doc/install-ubuntu.md]: Missing sudo

Comment (sieira) [inline doc/install-ubuntu.md]: the one in line 25?

File: doc/install-ubuntu.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    grunt test-unit-backend --test=test/unit-backend/webserver/index.js
+
+Note: This works for backend and midway tests.
+
+Some specialized Grunt tasks are available :
+
+    grunt linters # launch hinter and linter against the codebase
+    grunt test-frontend # only run the fontend unit tests
+    grunt test-unit-backend # only run the unit backend tests
+    grunt test-midway-bakend # only run the midway backend tests
```
Comment (heroandtn3) [inline doc/install-ubuntu.md]: Should be `grunt test-midway-bakend`

**Comment by sieira** [comment]
Ok, I forgot to create a new branch for my last commits... sorry

**Pull request merged**